.github/workflows/codeql.yml

```diff
@@ -0,0 +1,41 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "develop", "master" ]
+  pull_request:
+    branches: [ "develop" ]
+  schedule:
+    - cron: "30 8 * * 3"
```

Comment:
Do we want it to run on a schedule?

Reply:
The schedule is interesting. Theoretically, new vulnerabilities can be discovered over time so it may not be a bad idea for our less frequently touched repos 

https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/customizing-code-scanning#configuring-frequency